src/App.js

```diff
@@ -1,16 +1,41 @@
 import React from 'react';
 
+// Component setup part
+const todoList = [
+  {
+    id:1,
+    title:"Start the react Flamingo class."
+  },
+  {
+    id: 2,
+    title: "Complete assignment."
+  },
+  {
+    id: 3,
+    title: "Submit your firs lesson assignment."
+  },
+  {
+  id: 4,
+  title: "Submit all your lessons of Flamingo class."
+  }
+];
+
+
+// View part of react
 function App() {
   return (
-    <div style={{ textAlign: 'center' }}>
-      <header>
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
+    <div>
+      <h1>Todo List</h1>
+      <hr/>
+      {/* Show list items here */}
+      <ul>
+        {todoList.map(function(item) {
+          return (
+          <li key={item.id}>{item.title}</li>
+          )
+        }
+        )}
```

Comment:
Small nitpick for these lines (**doesn't require any changes**):

Typically, you'll see function expressions (fat arrow functions `() => {...}`) used _within_ React components. While they function the same as a traditional function within this context, they have some advantages in that they're **subjectively** a bit easier to write and read:
```jsx
<ul>
  {todoList.map(item => {
    return <li key={item.id}>{item.title}</li>
  })}
</ul>
```
Function expressions also have an advantage of [implicit returns](https://dev.to/samanthaming/es6-arrow-functions-cheatsheet-1cn). Therefore, the above could be simplified down to:
```jsx
<ul>
  {todoList.map(item => <li key={item.id}>{item.title}</li>)}
</ul>
```

Once again, this **doesn't require any changes** , so you're welcomed to keep your code as is.

Reply:
Hi Mr Matt. I understand you. I often used the arrow function, but for some reason, I forgot about it this time. It is elementary for me. Thanks for the hint.